internal/controller/nonadminbackupstoragelocation_controller.go

```diff
@@ -56,10 +59,13 @@ type NonAdminBackupStorageLocationReconciler struct {
 	Scheme          *runtime.Scheme
 	EnforcedBslSpec *velerov1.BackupStorageLocationSpec
 	OADPNamespace   string
+	SyncPeriod      time.Duration
```

Comment:
TODO user input

Reply:
I think the default could be minute, similar to Velero Sync ?

https://github.com/vmware-tanzu/velero/blob/6ac38cde8556f0fb338f1b00cd64746790b6c73c/pkg/controller/backup_sync_controller.go#L51

Reply:
Would it be useful to use sync period from velero sync here? So do not have separate sync for each, just one config that will be for both (existing velero sync one)

Reply:
yes that would make sense. Import the default value also check for velero server arg if they are set by the admin.
Then pass it on to NAC controller. We can do this via env var on NAC controller. WDYT ? @mateusoliveira43 @mpryc

Reply:
I like this idea if it's easy to implement.

Reply:
can not import, its unexported https://github.com/vmware-tanzu/velero/blob/6ac38cde8556f0fb338f1b00cd64746790b6c73c/pkg/cmd/server/config/config.go#L24